popup.css

```diff
@@ -101,18 +101,25 @@ border-radius: 5px;
     position: relative;
     min-height: 40px;
   }
-
+  
+  .vote {
+	font-size: 12px;
+	  color: #FFFFFF; 
+       left: 8%;	  
+  }
+  
   .ratings,
   .runtime {
     position: absolute;
     top: 50%;
     transform: translateY(-50%);
+	color:  #FFFFFF;
```

Comment:
Why is this needed? Computed color of this element is already white in the current one.

Reply:
Sorry, old habits. it's not needed.